datalab/datalab.sh

```diff
@@ -28,53 +38,65 @@ function update_apt_get() {
   return 1
 }
 
-ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
-PROJECT=$(/usr/share/google/get_metadata_value ../project/project-id)
-DOCKER_IMAGE=$(/usr/share/google/get_metadata_value attributes/docker-image || true)
-SPARK_PACKAGES=$(/usr/share/google/get_metadata_value attributes/spark-packages || true)
+function err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $@" >&2
+  return 1
+}
 
 if [[ "${ROLE}" == 'Master' ]]; then
```

Comment:
I noticed some of the other scripts are moving to using a `main` method at the bottom, and in `main` saying `if role = master run configure_master`. That might be nice here too.

E.g. https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/pull/191/files